src/server/index.js

```diff
@@ -1,12 +1,42 @@
+require('dotenv').config();
 const express = require('express');
-const cors = require('cors');
-const os = require('os');
+const session = require('express-session');
+const http = require('http');
+const passport = require('passport');
+const socketio = require('socket.io');
+const logger = require('morgan');
+const authRouter = require('./util/auth/auth.router');
+const passportInit = require('./util/auth/passport.init');
+const verifyEnv = require('./util/verify_env');
```

Comment:
Mixing `imports` and `require` seems like bad practice. Please also alphabetize

Reply:
So I agree, except I only use `import` in React files, and `require` in Express/back-end files. This is because `import` isn't actually a native Javascript feature, it comes from ES6, which we have Babel configured to handle for React. Most documentation online seems to use regular Javascript for the back-end, so this is actually something you see in other projects.

Reply:
Can we use “import” everywhere and make it work? I get that other projects do this, but I see no reason to cargo cult inconsistent style

Reply:
Would be a lot of work, and introduce (arguably unnecessary) new dependencies by half-porting our server code to ES6.

Reply:
The server is node/up to date JavaScript. Please use imports, it’s much more readable